src/app/components/publiclists/teamlist/teamlist.component.ts

```diff
@@ -269,46 +333,23 @@ export class TeamlistComponent implements OnInit {
    */
   fetchTeams(path) {
     const SELF = this;
+    this.globalService.startLoader('Fetching Teams');
```

Comment:
@sanketbansal when do you see this? I think I was not able to view it.

Reply:
Basically, it gets viewed when the loader starts on fetching data.
In your case it is fetching data locally very quick and that's why you might not be able to see it.